pkgs/games/zemeroth/default.nix

```diff
@@ -0,0 +1,105 @@
+{ stdenv
+, fetchFromGitHub
+, runCommand
+, rustPlatform
+, pkgconfig
+, makeWrapper
+, makeDesktopItem
+, alsaLib
+, libGL
+, libpthreadstubs
+, libSM
+, libudev
```

Comment:
maybe we can change this to just udev, libudev is an alias to udev since 2018-04-25

Reply:
Changed to udev.